drawer.html

```diff
@@ -0,0 +1,54 @@
+<div class="o-header__drawer" id="o-header-drawer" role="navigation" aria-label="Drawer menu" data-o-header-drawer data-o-header-drawer--no-js>
+	<div class="o-header__drawer-inner">
+
+		<div class="o-header__drawer-tools">
+			<a class="o-header__drawer-tools-logo" href="/">
+				<span class="o-header__visually-hidden">Financial Times</span>
+			</a>
+			<button type="button" class="o-header__drawer-tools-close" aria-controls="o-header-drawer" title="Close drawer menu">
+				<span class="o-header__visually-hidden">Close</span>
+			</button>
+		</div>
+
+		{{#if @root.licenceList.length }}
+			<div class="header__drawer-form-container">
+				<div class="o-forms licence-switcher__form">
+					<form action="/overview/licenceSwitch" method="POST" class="o-forms__affix-wrapper">
```

Comment:
This action needs to be configurable. So that the users page, for example, sends you to the users page, not overview.

Fancy giving it a go @kateloschinina ?